common/artifacts/000_wc_artifacts.txt

```diff
@@ -562,12 +670,190 @@ runeblade_unholy_1 = {
 	}
 
 	martial = 1
-	combat_rating = 5
+	combat_rating = 6
+	
+	monthly_character_prestige = 0.15
+	monthly_character_piety = 0.05
+
+	command_modifier = {
+		morale_offence = 0.05
+	}
 
 	picture = "GFX_runeblade_unholy"
 	slot = weapon
 }
 
+### One-handed weapon ###
+
+# Legendary
+felomelorn = {
+	quality = 5
+	flags = { combat_weapon sword }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 1
+	learning = 2
+	combat_rating = 10
+
+	monthly_character_prestige = 0.8
+	monthly_character_piety = 0.35
+	
+	general_opinion	= 5
+	cult_of_sunwell_opinion = 5
+
+	picture = "GFX_felomelorn"
+	slot = weapon
+}
+serathil = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	combat_rating = 25
+
+	monthly_character_prestige = 1.5
+	
+	orc_opinion = 10
+	blood_orc_opinion = 10			# fix this (trait blood_orc)
+
+	picture = "GFX_serathil"
+	slot = weapon
+}
+sulfuras = {
+	quality = 5
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		is_physical_class_trigger = yes
+		martial = 15
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 2
+	intrigue = 1
+	learning = 1
+
+	health = 0.5
+	combat_rating = 25
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+
+	picture = "GFX_sulfuras"
+	slot = weapon
+}
+hammer_of_the_twilight = {
+	quality = 5
+	flags = { combat_weapon hammer }
+
+	active = {
+		is_adult = yes
+		OR = {
+				is_magic_class_trigger = yes
+				is_physical_class_trigger = yes
+		}
+		religion = old_gods_worship
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
```

Comment:
If this thing is unique, then why you can craft multiple?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/66810046-2b49b000-ef37-11e9-8b49-a1bbac0ef95a.png)
![image](https://user-images.githubusercontent.com/46576860/66810070-31d82780-ef37-11e9-8588-326e5e32d288.png)

</details>

Reply:
@sejtalk please respond. you did decision.

Reply:
not craftable anymore